### DISCUSSION TITLE
Data Import

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5377] [Q&A] [answered]
I already setup the data import addon but i need to handle it via permissions so i can set it to some users and others not as it appears now as public for any user. How can i handle this ?

**Comment by duncanmcclean** [comment, marked as answer]
Looking at the code for the 'Data Import' addon, it doesn't look like it has any Permissions stuff setup. As this is a third party addon, please open an issue on the [addon's GitHub repository](https://github.com/riasvdv/statamic-data-import/issues).

**Reply by HazemMohamed252** [reply]
OK thanks for your support
